### PR TITLE
[stable/artifactory-ha]Fixed ingress and added support for TLS cert and PVC in nginx

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ $ helm install --name artifactory-ha --set artifactory.license.secret=artifactor
 **NOTE:** You have to keep passing the license secret parameters as `--set artifactory.license.secret=artifactory-cluster-license,artifactory.license.dataKey=art.lic` on all future calls to `helm install` and `helm upgrade`!
 
 #### Scaling your Artifactory cluster
-A key feature in Artifactory HA is the ability to set an initial cluster size with `--set artifactory.node.replicaCount=${CLUSTER_SIZE}` and if needed, resize it.
+A key feature in Artifactory HA ia the ability to set an initial cluster size with `--set artifactory.node.replicaCount=${CLUSTER_SIZE}` and if needed, resize it.
```
Comment (unguiculus) [inline stable/artifactory-ha/README.md]: This seems to be an accidental typo?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/artifactory-ha/README.md (hunk 1 above)
Comment (jainishshah17) [reply stable/artifactory-ha/README.md]: done

File: stable/artifactory-ha/templates/nginx-deployment.yaml [hunk 2]
```diff
@@ -46,11 +60,36 @@ spec:
...
+        - name: ssl-secret-volume
+          mountPath: "/var/opt/jfrog/nginx/ssl"
+        {{- end }}
         resources:
 {{ toYaml .Values.nginx.resources | indent 10 }}
+      volumes:
+      - name: nginx-volume
+      {{- if .Values.nginx.persistence.enabled }}
+        persistentVolumeClaim:
+          claimName: {{ .Values.artifactory.persistence.existingClaim | default (include "artifactory-ha.nginx.fullname" .) }}
```
Comment (eldada) [inline stable/artifactory-ha/templates/nginx-deployment.yaml]: Change `artifactory.persistence.existingClaim` to `nginx.persistence.existingClaim`

Comment (jainishshah17) [reply stable/artifactory-ha/templates/nginx-deployment.yaml]: done

**Review by unguiculus — APPROVED** [review]

**Pull request merged**